processor/processor.go

```diff
@@ -1703,10 +1707,13 @@ func (proc *Handle) preprocessStage(partition string, subJobs subJob) (*preTrans
 			proc.logger.Debugn("Missing traceParent in processJobsForDest", logger.NewIntField("jobId", batchEvent.JobID))
 		} else {
 			ctx := stats.InjectTraceParentIntoContext(context.Background(), traceParent)
-			_, span = proc.tracer.Start(ctx, "proc.processJobsForDest", stats.SpanKindConsumer, stats.SpanWithTags(stats.Tags{
-				"workspaceId": batchEvent.WorkspaceId,
-				"sourceId":    eventParams.SourceId,
-			}))
+			_, span = proc.tracer.Trace(ctx, "processJobsForDest", tracing.WithTraceKind(stats.SpanKindConsumer),
```

Comment:
need to change the name `processJobsForDest` to something else?

Reply:
Good catch :+1: